.gitignore

```diff
@@ -6,3 +6,6 @@
 # Datasource local storage ignored files
 /dataSources/
 /dataSources.local.xml
+/BudgetBuddy/target
+/BudgetBuddy/target
+/.idea
```

Comment:
Creating a gitignore is very important. This prevents unnecessary files from being uploaded to github that take up more storage, and can cause issues down the line.
https://martinfowler.com/bliki/TechnicalDebt.html